client/app/bundles/course/experience-points/disbursement/components/forms/PointTextField.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC } from 'react';
+import { Control, Controller } from 'react-hook-form';
+import { DisbursementFormData } from 'types/course/disbursement';
+import FormTextField from 'lib/components/form/fields/TextField';
+
+interface Props {
+  control: Control<DisbursementFormData>;
+  index: number;
+  fieldId: string;
+  setValue: (
+    name:
+      | 'reason'
+      | 'currentGroup'
+      | 'pointList'
+      | `pointList.${number}`
+      | `pointList.${number}.id`
+      | `pointList.${number}.points`,
+    value: string | number,
+  ) => void;
+  className?: string;
+}
+
+const PointTextField: FC<Props> = (props: Props) => {
+  const { control, index, fieldId, setValue, className } = props;
+  return (
+    <Controller
+      name={`pointList.${index}.points`}
+      control={control}
+      render={({ field, fieldState }): JSX.Element => (
+        <FormTextField
+          field={field}
+          fieldState={fieldState}
+          // @ts-ignore: component is still written in JS
+          fullWidth
+          InputLabelProps={{
+            shrink: true,
+          }}
+          onWheel={(event): void => event.currentTarget.blur()}
+          onChangeCustom={(value): void => {
```

Comment:
Just checking whats the reason for this?

Reply:
I made it so that it should set the value only if it is a number that is keyed in, but it doesnt seem to work on firefox so I have removed it.